assets/js/googlesitekit-user.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Entrypoint for the "core/user data store.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+
```

Comment:
This is included in some other files, mostly as a hack to satisfy `@wordpress/eslint-plugin` for the `dependency-groups` rule because it couldn't understand our aliases. That isn't a problem anymore (because we got rid of them) so this can be removed 👍